packages/grafana-ui/src/components/Logs/LogDetailsRow.tsx

```diff
@@ -120,6 +119,17 @@ class UnThemedLogDetailsRow extends PureComponent<Props, State> {
         </div>
         <div className={style.logsRowCell}>
           <span>{parsedValue}</span>
+          {links &&
+            links.map(link => {
+              return (
+                <span key={link}>
+                  &nbsp;
+                  <a href={link}>
+                    <i className={'fa fa-external-link'} />
+                  </a>
+                </span>
```

Comment:
The context menu equivalent of this is:

https://github.com/grafana/grafana/blob/master/packages/grafana-ui/src/components/ContextMenu/ContextMenu.tsx#L218

Reply:
Do you mean the icon or the whole style? Probably need to sync with @davkal on this. Right now this looks like this:
<img width="624" alt="Screenshot 2019-11-02 at 00 03 21" src="https://user-images.githubusercontent.com/1014802/68061256-39077f80-fd04-11e9-9e40-a1e073eef7d1.png">

And the issue is we have hover on each line with color change (the yellow row). Which makes sense as it helps to pick up line visually. This means clickability is not apparent there so we need something to make it different from all other lines. Also there can be multiple links so there can be multiple icons (thought that is not great UI, I think that should not be common). So we could use context menu instead as in panels to show the links but we would still need something to make the field in the detail look clickable and also probably add some kind of label to the link config in the datasource configuration page.

Reply:
Does it makes sense to do do stats on traces (button with fa-signal icon)? Wouldn't it make more sense to replace stats button with link icon (fa-external-link) and have there link to the trace? Or if it makes sense, we could just add another link button (similarly as filter/filter out buttons). Also, we were talking with @davkal that derived fields would have the same heading as Parsed fields and  Log Labels, so it is easier to distinguish between everything.

Reply:
> Does it makes sense to do do stats on traces (button with fa-signal icon)?

Hmm honestly I assume yes (though I need to fix that code it seems). Putting the link there would make sense but depends on how we want to handle multiple links then.

> Also, we were talking with @davkal that derived fields would have the same heading as Parsed fields and Log Labels, so it is easier to distinguish between everything.

Not sure in case traceID is in the message only thing we do more than normal parsing is that we add link. So in case of no derived fields you would have traceId in parsed field without link basically. With derived fields you would have same traceId just with the link and in different section which may seem weird.

Reply:
@davkal 
> +1 on putting derived fields into their own section, both for discoverability (config section had that name) and looking forward where some log formats won't have parsed fields because there is no parser.

If I get a `traceId` that is both from parsing and from derived field (same name, same value) is it ok to have it just in derived fields section then? Or maybe have them in both sections as kind of consistency?

> +1 on context menu in logs panel, where log details are not available

I think the question was more about how to present the link here in this detail. Whether it is ok to have icon separately as it is right now, with theoretically having more than 1. Moving the icon somewhere else, or having a context menu also here to show multiple links there.

Reply:
> If I get a traceId that is both from parsing and from derived field (same name, same value) is it ok to have it just in derived fields section then? Or maybe have them in both sections as kind of consistency?

I'd expect them in both sections, but I would not be confused if it only appeared in derived fields.